pkg/service/manifest/manifest.go

```diff
@@ -163,3 +164,136 @@ func (s Service) DeleteManifest(request DeleteManifestRequest) error {
 
 	return nil
 }
+
+// TODO: (Neal) Add entire validation framework in place of these validation checks
+func isValidApplication(s Service, request CreateApplicationRequest, ps exchange.PresentationSubmission) error {
+
+	gotManifest, err := s.storage.GetManifest(request.ManifestID)
+
+	if err != nil {
+		return util.LoggingErrorMsg(err, "problem with retrieving manifest during application validation")
+	}
+
+	if gotManifest == nil {
+		return util.LoggingNewError(fmt.Sprintf("application is not valid. A manifest does not exist with id: %s", request.ManifestID))
+	}
+
+	inputDescriptors := gotManifest.Manifest.PresentationDefinition.InputDescriptors
+	inputDescriptorIds := map[string]bool{}
```

Comment:
nit: for map initializations use make https://github.com/uber-go/guide/blob/master/style.md#initializing-maps
`inputDescriptorIDs = make(map[string]bool)`